packages/nuekit/test/nuekit.test.js

```diff
@@ -133,8 +133,8 @@ test('content collection', async () => {
   expect(actual).toEqual([
     { url: '/blog/first-a.html', title: 'First', dir: 'blog', slug: 'first-a.html' },
```

Comment:
```suggestion
    { url: '/blog/first-c.html', title: 'Third', dir: 'blog', slug: 'first-c.html' },
    { url: '/blog/first-a.html', title: 'First', dir: 'blog', slug: 'first-a.html' },
```

To fulfill `undefined` state at top of results.